spec/lib/rspec/fortify_spec.rb

```diff
@@ -208,5 +208,21 @@
         end
       end
     end
+
+    context 'when git diff fallback' do
```

Comment:
So the `STUB_GIT_DIFF` var is necessary here because we rely on `cmd.run!` and any local `allow()`/`receive()` stubs won't carry through to the new process, right?

Reply:
correct